src/test/scala/com/horizon/exchangeapi/TestDBConnection.scala

```diff
@@ -0,0 +1,36 @@
+package com.horizon.exchangeapi
```

Comment:
Now that we have this class, we should make an issue to apply this class to all new test classes. In particular, we should comment out the route in /admin suites that cleans out the changes table and pull those tests out into a new class that can use this functionality instead of a pretty unsafe route.